src/caret_analyze/architecture/architecture.py

```diff
@@ -381,6 +409,19 @@ def insert_publisher_callback(self, node_name: str,
 
     def insert_variable_passing(self, node_name: str,
                                 callback_name_write: str, callback_name_read: str) -> None:
+        """
+        Insert variable_passing in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node
+        callback_name_write : str
+            name of write callback in variable_passing to insert
+        callback_name_read : str
+            name of read callback in variable_passing to insert
```

Comment:
```suggestion
            name of write callback to be inserted in variable_passing
        callback_name_read : str
            name of read callback to be inserted in variable_passing
```

Reply:
Fix it in [6cf0285](https://github.com/tier4/CARET_analyze/pull/300/commits/6cf0285fa738613d5ed20fe836eed60ce682205c).